### DISCUSSION TITLE
data-wp-text shows <br>, not parsed as break

### DISCUSSION THREAD

**Discussion by webconstructor** [WordPress/gutenberg#63699] [Interactivity API]
how can I have html tags be rendered like html in 
`<h3 data-wp-text="context.question"></h3>`

**Comment by sirreal** [comment]
The text directive is for text, HTML is not supported.

There's a mention of `data-wp-dangerous-html` in this discussion: https://github.com/WordPress/gutenberg/discussions/52904 It doesn't exist now, but that's the feature you're looking for.

If you have trusted HTML to inject, the workaround at this time is to [set `innerHTML`](https://developer.mozilla.org/en-US/docs/Web/API/Element/innerHTML) via DOM APIs. There's no directive at this time. You should be _very careful_ when doing this.
